rules/windows/powershell/powershell_module/posh_pm_remotefxvgpudisablement_abuse.yml

```diff
@@ -0,0 +1,31 @@
+title: Potential RemoteFXvGPUDisablement.EXE Abuse - PowerShell Module
+id: 38a7625e-b2cb-485d-b83d-aff137d859f4
+related:
+    - id: a6fc3c46-23b8-4996-9ea2-573f4c4d88c5 # ProcCreation
+      type: similar
+    - id: f65e22f9-819e-4f96-9c7b-498364ae7a25 # PS Classic
+      type: similar
+    - id: cacef8fc-9d3d-41f7-956d-455c6e881bc5 # PS ScriptBlock
+      type: similar
+status: experimental
+description: Detects PowerShell module creation where the module Contents are set to "function Get-VMRemoteFXPhysicalVideoAdapter". This could be a signe of potential abuse of "RemoteFXvGPUDisablement.exe" binary which is known to be vulnerable to module load-order hijacking.
```

Comment:
```suggestion
description: Detects PowerShell module creation where the module Contents are set to "function Get-VMRemoteFXPhysicalVideoAdapter". This could be a sign of potential abuse of the "RemoteFXvGPUDisablement.exe" binary which is known to be vulnerable to module load-order hijacking.
```